### PR TITLE
enforcing `controller` type to be a list of string

### PR DESCRIPTION
**Pull request by jiyoontbd**
blocks https://github.com/TBD54566975/web5-kt/pull/248

### CODE REVIEW

File: test-vectors/did_jwk/resolve.json [hunk 1]
```diff
@@ -7,10 +7,7 @@
       "output": {
         "@context": "https://w3id.org/did-resolution/v1",
         "didDocument": {
-          "@context": [
-            "https://www.w3.org/ns/did/v1",
-            "https://w3id.org/security/suites/jws-2020/v1"
-          ],
+          "@context": "https://www.w3.org/ns/did/v1",
```
Comment (jiyoontbd) [inline test-vectors/did_jwk/resolve.json:10]: forcing @context to be a string, not a list of strings

File: test-vectors/did_dht/create.json [hunk 2]
```diff
@@ -88,7 +88,9 @@
       },
       "output": {
         "id": "did:dht:cyuoqaf7itop8ohww4yn5ojg13qaq83r9zihgqntc5i9zwrfdfoo",
-        "controller": "did:example:abcd",
+        "controller": [
```
Comment (jiyoontbd) [inline test-vectors/did_dht/create.json:91]: forcing controller to be a list of strings

File: test-vectors/did_jwk/resolve.json (hunk 1 above)
Comment (decentralgabe) [reply test-vectors/did_jwk/resolve.json:10]: could you force this to be a list of strings? I think that will be the most common case for context (if we should choose to use it)

Comment (jiyoontbd) [reply test-vectors/did_jwk/resolve.json:10]: sure, that sounds good! i'll update the original PR to build DIDDocument with a list of string for `@context` now.

**Review by decentralgabe — APPROVED** [review]

**Review by mistermoe — APPROVED** [review]

**Pull request merged**